src/page/auth.ejs

```diff
@@ -19,16 +19,16 @@
 
     <title><%= BRAND_NAME %></title>
 
-    <script src="../min/basicBrowserFeatureCheck.js?<%= VERSION %>"></script>
-    <script src="../min/checkBrowser.js?<%= VERSION %>"></script>
+    <script src="../min/basicBrowserFeatureCheck.js?v=<%= VERSION %>"></script>
+    <script src="../min/checkBrowser.js?v=<%= VERSION %>"></script>
```

Comment:
in the HTML files we manually add this `v` parameter that will allow the cache to be invalidated as soon as a new version is available.